prog-o-meter.py

```diff
@@ -277,6 +283,26 @@ def read_days_file(_filename):
     days_text.close() 
     return days
 
+def get_completion_date(days_remaining):
+    """Calculate the date at which the challenge will be over.
+
+    Args:
+        days_remaining: number of days remaining in the project
+
+    Returns:
+        The project completion date as a string
+    """
+    
+    today = datetime.date.today()
+    completion_date = today + datetime.timedelta(days=days_remaining)

```

Comment:
You asked about whether this function should be a part of the ProgressGUI class. I think it would be a nice thing to do. The function is not used outside the class (as far as I can tell), so I think it belongs inside the class.

Reply:
Alright then :) I'll do that and see how it goes.

Reply:
Cool :) Thanks! 
Let me know if you run into problems! I look forward to merging in this addition